tigerpath/majors_and_certificates/majors/COS-AB_2018.json

```
@@ -182,9 +183,9 @@
                 "COS 314/MUS 314: Computer and Electronic Music through Programming, Performance, and Composition",
                 "COS 323/ORF 363: Computing and Optimization for the Physical and Social Sciences",
                 "COS 324: Introduction to Machine Learning",
-                "COS 325/MUS 315: Transforming Reality by Computer",
                 "COS 326: Functional Programming",
                 "COS 343: Algorithms for Computational Biology",
+                "COS 360/PSY 360: Computational Models of Cognition",
                 "COS 401/TRA 301: Introduction to Machine Translation ",
```

```suggestion
                 "COS 401/TRA 301: Introduction to Machine Translation",
```
Just, if we're already fixing it in one, then we might as well fix it in the other.